composer.json

```diff
@@ -10,7 +10,7 @@
     "drupal/queue_mail": "^1",
     "drupal/jwt": "^1.0",
     "drupal/config_split": "^2.0",
-    "dpc-sdp/bay_platform_dependencies": "^2.0.2"
+    "dpc-sdp/bay_platform_dependencies": "^2.0.3"
```

Comment:
don't think you will need this release. cause the version is not locked, so it should pull the latest one. Try running it for health and see how it turns out.

Reply:
Thanks Nadim, agree.